examples/private_generics/integration-tests/private_generics.move

```diff
@@ -0,0 +1,12 @@
+//# init --addresses test=0x42
```

Comment:
The test cases should be placed in the `crates/rooch-framework-tests` directory.

Reply:
I'll add them in the other PR: https://github.com/rooch-network/rooch/pull/285

Reply:
The functionality of private_generics, which was tested in the integration testing, has been completed. Please make the necessary code modifications and resubmit.

Reply:
@steelgeek091 it didn't work after pulling the latest.

Command: 

```zsh
rooch move integration-test -p examples/private_generics/ --named-addresses rooch_examples=0x34141 --ub
```

Result:

```zsh
Documentation generation successful!
precompiled module: None, 0x34141, "Test1"
precompiled module: None, 0x34141, "Test2"
precompiled module: None, 0x34141, "Test3"
test integration-test333::private_generics.move ... ok

test result: ok. 1 passed; 0 failed; 0 filtered out

Success
```

Code:

```move
module rooch_examples::Test3 {
    use rooch_examples::Test1::Data;
    use rooch_examples::Test1::Box;
    use rooch_examples::Test1::new_data;
    use rooch_examples::Test1::create_box;
    use rooch_examples::Test1::box_value;

    public fun run() {
        let data = new_data(789);

        // Here, it will report that `Data` is not defined within the current module,
        // because `Data` is imported from the `Test1` module.
        let box_val = create_box<Data, Data, Box<u32>>(data);

        let _ = box_value(&box_val);
    }
}
```

Exp:

```
processed 2 tasks

task 1 'run'. lines 3-15:
status EXECUTED
```

Reply:
There are some erroneous debug print statements that I will delete later. 
Additionally, there is an issue with not printing the execution status of tasks. I will investigate the cause of this problem.

Reply:
> There are some erroneous debug print statements that I will delete later.
> Additionally, there is an issue with not printing the execution status of tasks. I will investigate the cause of this problem.

I rebuilt the command.

Now it can print. But the format is somewhat strange:

```exp
error: resource type "Data" in function "0x34141::Test1::create_box" not defined in current module or not allowed
   ┌─ /tmp/tempfile.move:62:23
   │
62 │         let box_val = create_box<Data, Data, Box<u32>>(data);
   │                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^


processed 4 tasks

task 1 'run'. lines 3-10:
status EXECUTED

task 2 'run'. lines 12-19:
status EXECUTED

task 3 'run'. lines 21-29:
status EXECUTED
```

Reply:
@steelgeek091 needs to resolve above printing issue in this PR when executing `rooch move integration-test`

Reply:
Is above issue resolved?